### PR TITLE
add `stylelint-define-config`

### PR DESCRIPTION
**Pull request by pengzhanbo**
> Which issue, if any, is this issue related to?

None, submit a new tool to provide stylelint config's types support.

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,6 +141,7 @@
 
 - [pre-commit-stylelint](https://github.com/thibaudcolas/pre-commit-stylelint) - Mirrors all releases for the pre-commit hooks framework.
 - [stylelint-find-new-rules](https://github.com/Donov4n/stylelint-find-new-rules) - Find Stylelint rules that are not configured in your Stylelint config.
+- [stylelint-define-config](https://github.com/stylelint-types/stylelint-define-config) - Provide a defineConfig function for stylelint.config.js files
```
Comment (ybiquitous) [inline README.md:144]: [suggest]

- Sort alphabetically.
- Add a missing period.
- Use backslashes.

```suggestion
- [stylelint-define-config](https://github.com/stylelint-types/stylelint-define-config) - Provide a `defineConfig` function for `stylelint.config.js` files.
- [stylelint-find-new-rules](https://github.com/Donov4n/stylelint-find-new-rules) - Find Stylelint rules that are not configured in your Stylelint config.
```

**Review by ybiquitous — CHANGES_REQUESTED** [review]
@pengzhanbo Thanks for the pull request. It's an amazing package!

I've left a comment about document styling. Can you deal with it, please?

File: README.md (hunk 1 above)
Comment (pengzhanbo) [reply README.md:144]: Thank you for pointing out and fixing the styling issues!

**Review by ybiquitous — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**